jf_agent/main.py

```diff
@@ -5,19 +5,20 @@
 import urllib3
 import yaml
 import dateparser
-from datetime import datetime
 import pytz
+from datetime import datetime
```

Comment:
unused?

Reply:
Ah, the diff is misleading here. I just moved the import line, this is referenced in `line 76`